inc/namespace.php

```diff
@@ -29,6 +29,12 @@ function bootstrap() {
 		Block_Editor\bootstrap();
 	}
 
+	if ( $config['xmlrpc'] === false ) {
+		add_filter( 'xmlrpc_enabled', __NAMESPACE__ . '\\filter_xmlrpc_handler', 999 );
+		add_filter( 'xmlrpc_methods', __NAMESPACE__ . '\\filter_xmlrpc_methods_handler', 999 );
+		add_filter( 'xmlrpc_element_limit', __NAMESPACE__ . '\\filter_xmlrpc_element_limit_handler', 999 );
```

Comment:
I'm curious why all of these filters are needed and not just the `xmlrpc_enabled` one?

Reply:
There's a comment in core:
```
 		 * Contrary to the way it's named, this filter does not control whether XML-RPC is *fully*
		 * enabled, rather, it only controls whether XML-RPC methods requiring authentication - such
		 * as for publishing purposes - are enabled.
```

Reply:
Wow well that's fun 🤦‍♂ 
I did spot a potential nuclear option too as well if we want to go whole hog:

```php
$wp_xmlrpc_server_class = apply_filters( 'wp_xmlrpc_server_class', 'wp_xmlrpc_server' );
$wp_xmlrpc_server       = new $wp_xmlrpc_server_class;

// Fire off the request
$wp_xmlrpc_server->serve_request();
```

We could add a fake XMLRPC class that only has an empty `serve_request()` method that either does nothing or returns a 404 page.

Reply:
@rmccue would this option be preferable?

Reply:
For reference I think what you have here with these three filters is fine right now, I jjust found it interesting. The above suggestion would require a bunch of testing as it could affect compatibility with some plugins.